<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelegraphWindow</class>
 <widget class="QMainWindow" name="TelegraphWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Telegraph</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QFrame" name="contactsFrame">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QStackedWidget" name="contactsSW">
         <widget class="QWidget" name="contactsPage">
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QPushButton" name="profilePB">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="searchLE">
             <property name="placeholderText">
              <string>Search</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QListWidget" name="contactsLW">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>480</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="lineWidth">
              <number>1</number>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="spacing">
              <number>3</number>
             </property>
             <property name="selectionRectVisible">
              <bool>false</bool>
             </property>
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="profilePage">
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="userLabel">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(114, 159, 207);</string>
             </property>
             <property name="text">
              <string>Username:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="phoneLabel">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(114, 159, 207);</string>
             </property>
             <property name="text">
              <string>Phone:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="backPB">
             <property name="text">
              <string>&lt;-</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="messagesFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QStackedWidget" name="messagesSW">
         <widget class="QWidget" name="emptyPage">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="selectLabel">
             <property name="text">
              <string>Please Select a Chat to Start Messaging</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="messagesPage">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="2" column="1">
            <widget class="QPlainTextEdit" name="messagePTE">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="sendPB">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>-&gt;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QTableWidget" name="messagesTW">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="showGrid">
              <bool>false</bool>
             </property>
             <property name="columnCount">
              <number>2</number>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>150</number>
             </attribute>
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>150</number>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column/>
             <column/>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QLineEdit" name="searchMessageLE">
             <property name="placeholderText">
              <string>Search</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
